doc/user/tutorials/uploader-tool/index.md

```diff
@@ -30,6 +30,12 @@ For complete transparency, the installer will install several components require
 * PyLauncher for Windows (<https://bitbucket.org/vinay.sajip/pylauncher>)
 * The IRIDA MiSeq Uploader source code (<https://github.com/phac-nml/irida-miseq-uploader>)
 
+### Command Line Uploader
+
+Development is being done on a new Command Line Uploader.
+
+If you would prefer using the command line to upload data, you can download it here <https://github.com/phac-nml/irida-uploader>
```

Comment:
Can you add a note that the tutorial expects you're using the GUI uploader?  I know the existing docs don't even show the current version of the uploader.  We'll have to update the tutorial once we're more settled with the new uploader.

Reply:
Good idea. Added.